xgboost_synthetic/util.py

```diff
@@ -11,6 +11,8 @@
 def notebook_setup():
   # Install the SDK
 
+  for p in ["sklearn", "pandas", "joblib", "fire"]:
```

Comment:
xgboost_synthetic has a requirements.txt file that lists all the dependencies. Could we do pip3 install -r requirements.txt.

that way going forward only the requirements.txt needs updates.

Reply:
These packages have been included in requirements.txt.
This change is to help us be able to run it in a stock notebook image that ships with Kubeflow.

Reply:
Correct. I meant that instead of iterating over these in the notebook, if we can pip3 install -r requirements.txt in here, that would future proof this. i.e. if new dependencies are added to requirements.txt we won't need to modify the code here.

Reply:
Oh, I got it. I will fix it.